README.md

```diff
@@ -13,50 +13,23 @@ For the current Java Sync driver reference documentation, see the
-
-## Build Instructions
-
-:warning: **When building docs for a new version, update the following items:**
-- `<this repo>/reference/config.toml` file to point to the new base version
-- `<this repo>/landing/data/releases.toml` file to link to the docs for the new version
-- `<thid repo>/reference/data/mongodb.toml` file to display the mew version
-- `<reference/content/driver-reactive/getting-started/installation.md` and `<reference/content/driver-scala/getting-started/installation.md` files to display the new version
-- Update the "Previous Versions" sections for the Scala and Java RS driver docs which are in ``reference/content/<driver name directory>/index.md``
-- Meta redirects as shown in this PR [Java sync docs redirect](https://github.com/mongodb/docs-java-other/pull/3/files#diff-0f1a8692163867f83ff7451f3018bae71d3d16dbee396abf03263784e5dda940)
-
-After updating the documentation, submit a pull request for approval.
-
-Once your PR is approved and merged, pull the latest changes.
-Then run the `publish-docs` script with the version using the
-format `<major>.<minor>`:
-
-```sh
-./publish-docs <version, e.g. 4.7>
-```
-
-This command updates the submodule that tracks the *gh-pages* branch
-of the [mongo-java-driver](https://github.com/mongodb/mongo-java-driver).
-Then it builds the documentation in a new directory that corresponds to the
-new version name.
+**Now that Java RS and Scala documentation is built on Snooty, you do not
+have to update the legacy documentation site.**
 
 ## Building API Documentation
 
 Always build the API docs for any new major and minor releases.
 
-To build the API docs, navigate to your `mongo-java-driver` repo 
-(*Note: NOT the submodule in this repo*) and executing the appropriate `gradlew` command
-after checking out the correct tag. Ensure you installed the Java version specified in the `:bson:compileJava` task in your development environment prior to building.
+To build the API docs, navigate to your `mongo-java-driver` repo
+(_Note: NOT the submodule in this repo_) and execute the appropriate `gradlew` command
+after checking out the correct tag. Ensure you installed the Java
+version specified in the `:bson:compileJava` task in your development
+environment prior to building. In the most recent (5/2024) deploy, this
+was Java 17.
 
-:warning: **Use the GitHub "release" that correspond to the version of the driver rather than a branch**
+:warning: **Use the GitHub "release" that corresponds to the version of the driver rather than a branch**
 
-For example, to build the API docs for the 4.4 release of driver run the following commands:
+For example, to build the API docs for the 4.4 release of the driver run the following commands:
 
 ```sh
 git checkout r4.4.0
```

Comment:
I: there was one extra step I had to take before copying the `build/docs` folder! Since you're not running the publish-docs script from the old build instructions, you need to create the `<version>/apidocs` directory yourself before copying the docs files. 

I think we could add a step that instructs the user to navigate to `docs-java-other/mongo-java-driver` and run `mkdir -p <version>/apidocs`. Then, they can run `cp -a ../mongo-java-driver/build/docs ./mongo-java-driver/<version>/apidocs`